inlang/source-code/website/src/services/auth/src/onSignOut.ts

```diff
@@ -1,13 +1,28 @@
 import type { SetStoreFunction } from "solid-js/store"
 import type { LocalStorageSchema } from "../../../services/local-storage/index.js"
+import { publicEnv } from "@inlang/env-variables"
 import { telemetryBrowser } from "@inlang/telemetry"
 
 /**
  * This function is called when the user clicks the "Sign Out" button.
  */
 export async function onSignOut(args: { setLocalStorage: SetStoreFunction<LocalStorageSchema> }) {
 	// sign out on the server
-	await fetch("/services/auth/sign-out", { method: "POST" })
+	const hasExternalGitProxy =
+		publicEnv.PUBLIC_SERVER_BASE_URL !== publicEnv.PUBLIC_GIT_PROXY_BASE_URL
+
+	if (hasExternalGitProxy) {
+		await Promise.allSettled([
+			fetch("/services/auth/sign-out", { method: "POST" }),
```

Comment:
is the auth sign out API route (telefunc) still required? Couldn't the code here simply set the local storage['user'] to undefined?

Reply:
@samuelstroschein  no as the cookie needs to be deleted which is http only.